packages/components/src/navigator/navigator-screen/component.tsx

```diff
@@ -53,30 +65,26 @@ function UnconnectedNavigatorScreen(
 		return () => removeScreen( screen );
 	}, [ screenId, path, addScreen, removeScreen ] );
 
-	const isRTL = isRTLFn();
-	const { isInitial, isBack } = location;
+	// Animation.
+	const { animationStyles, onScreenAnimationEnd, shouldRenderScreen } =
+		useScreenAnimatePresence( {
+			isMatch,
+			isBack,
+			skipAnimation: skipAnimationAndFocusRestoration,
+		} );
+
+	// Styles
 	const cx = useCx();
 	const classes = useMemo(
-		() =>
-			cx(
-				styles.navigatorScreen( {
-					isInitial,
-					isBack,
-					isRTL,
-				} ),
-				className
-			),
-		[ className, cx, isInitial, isBack, isRTL ]
+		() => cx( styles.navigatorScreen, animationStyles, className ),
+		[ className, cx, animationStyles ]
 	);
 
+	// Focus restoration
```

Comment:
There are no runtime changes re. focus restoration, only variable renaming and some prop destructuring that happened outside the `useEffect` hook